lib/UI/views/stream_example/stream_example_view.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:RMFlutter/UI/views/stream_example/stream_example_view_model.dart';
+import 'package:flutter/material.dart';
+import 'package:stacked/stacked.dart';
+
+class StreamExampleView extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return ViewModelBuilder<StreamExampleViewModel>.reactive(
+      builder: (context, model, child) => Scaffold(
+        body: Center(
+          child: Text(model.title),
+        ),
+        floatingActionButton: MaterialButton(
+          child: Text('Change Stream Srouces'),
+          onPressed: model.swapSources,
+        ),
+      ),
```

Comment:
Whoops, I missed one :eyes: 

```suggestion
      builder: (context, viewModel, child) => Scaffold(
        body: Center(
          child: Text(viewModel.title),
        ),
        floatingActionButton: MaterialButton(
          child: Text('Change Stream Srouces'),
          onPressed: viewModel.swapSources,
        ),
      ),
```